uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/feature/ImplicitGrantIT.java

```diff
@@ -62,34 +59,13 @@
     @Autowired @Rule
     public IntegrationTestRule integrationTestRule;
 
-    @Autowired
-    WebDriver webDriver;
 
     @Value("${integration.test.base_url}")
     String baseUrl;
 
-    @Value("${integration.test.app_url}")
-    String appUrl;
-
     @Autowired
     RestOperations restOperations;
 
-    @Autowired
-    TestClient testClient;
-
-    @Before
-    @After
-    public void logout_and_clear_cookies() {
```

Comment:
What is the purpose of this change?

Reply:
Before I wrote the unit tests, I was considering writing an integration test. This is one of the classes that I considered writing the integration test in. I didn't understand the purpose of various members of this class, like these, and then found that in fact they were unused.
Hence I removed them to save confusion for the next developer who will read this class.

Reply:
Please **revert this change** even if it is useful. GE does not want to have any differences from the Pivotal codebase, unless they are related to some GE-specific features.

If you want, you can make a separate PR for this file directly against Pivotal UAA. If they approve it, then we will get the changes during the next version upgrade.

Reply:
Please **revert this change** even if it is useful. GE does not want to have any differences from the Pivotal codebase, unless they are related to some GE-specific features.

If you want, you can make a separate PR for this file directly against Pivotal UAA. If they approve it, then we will get the changes during the next version upgrade.

Reply:
OK